packages/flutter/lib/src/widgets/notification_listener.dart

```diff
@@ -53,10 +53,10 @@ abstract class Notification {
   ///
   /// The notification will be delivered to any [NotificationListener] widgets
   /// with the appropriate type parameters that are ancestors of the given
-  /// [BuildContext].
+  /// [BuildContext]. If the [BuildContext] is null, the notification is
```

Comment:
Although "dropped on the floor" is well understood (and kind of funny) it  might be better to say "then the notification is not dispatched".